src/firmware/README.md

```diff
@@ -1,12 +1,34 @@
 # Fimware
 
-### gps_firmware
+### Udev Rules
```

Comment:
We should add an entry for this in the table of contents at the top of the `README` under `Arduino Development`. (We may also want to consider renaming `Arduino Development` to something like `Firmware Development`)

Reply:
I think "Udev Rules" entry is in the file under the `firmware folder` which is separate from the README in the main directory. In this case, there is no `Arduino Development` heading.

Reply:
Good point! :+1: